lib/elixir/scripts/windows_installer/installer.nsi

```diff
@@ -45,20 +52,22 @@ Function CheckOTPPageShow
     ${If} $0 == 0
       StrCpy $InstalledOTPRelease $1
       ${If} $InstalledOTPRelease == ${OTP_RELEASE}
-        ${NSD_CreateLabel} 0 0 100% 20u "Found existing Erlang/OTP $InstalledOTPRelease installation at $OTPPath. Please proceed."
+        ${NSD_CreateLabel} 0 0 100% 60u "Found existing Erlang/OTP $InstalledOTPRelease installation at $OTPPath. Please proceed."
+        StrCpy $OTPVerified "true"
       ${ElseIf} $2 < ${OTP_RELEASE}
-        ${NSD_CreateLabel} 0 0 100% 30u "Found existing Erlang/OTP $InstalledOTPRelease installation at $OTPPath but this Elixir installer was precompiled for Erlang/OTP ${OTP_RELEASE}. \
-        We recommend checking if there is an Elixir version precompiled for Erlang/OTP $InstalledOTPRelease. Otherwise, proceed."
+        ${NSD_CreateLabel} 0 0 100% 60u "Found existing Erlang/OTP $InstalledOTPRelease installation at $OTPPath but this Elixir installer was precompiled for Erlang/OTP ${OTP_RELEASE}. \
+        $\r$\n$\r$\nYou can either search for another Elixir installer precompiled for Erlang/OTP $InstalledOTPRelease or download Erlang/OTP ${OTP_RELEASE} and install before proceeding."
+        ${NSD_CreateLink}  0 60u 100% 20u "Download Erlang/OTP ${OTP_RELEASE}"
+        Pop $DownloadOTPLink
+        ${NSD_OnClick} $DownloadOTPLink OpenOTPDownloads
```

Comment:
<img width="527" alt="image" src="https://github.com/elixir-lang/elixir/assets/76071/563bd020-05bb-497c-8094-a2b0677d3d26">

Reply:
What if it is a too old version that we don't support? Or maybe OTP 27 is out and we don't have an installer yet? Is there a way to check if the link exists?

Reply:
Something to add: you can run Elixir compiled for OTP 26 on OTP 27, but you cannot run Elixir compiled for OTP 26 on OTP 25. So maybe we can make that as part of the warning. If OTP is older than the current one, then downloading is a must, if newer, then current is fine.

Reply:
Yes I can change that but not sure about the wording. At the moment it is:

```
Found existing Erlang/OTP $InstalledOTPRelease installation at $OTPPath but this Elixir installer was precompiled for Erlang/OTP ${OTP_RELEASE}.

You can either search for another Elixir installer precompiled for Erlang/OTP $InstalledOTPRelease or download Erlang/OTP ${OTP_RELEASE} and install before proceeding.
```

and it is for both when installed OTP is higher or lower than what installer was compiled against.